web-client/src/presenter/computeds/updatedFilePetitionHelper.ts

```diff
@@ -20,9 +21,14 @@ interface IOtherContactNameLabel {
 }
 
 type UpdatedFilePetitionHelper = {
-  filingOptions: string[];
+  filingOptions: { label: string; value: string }[];
+  getLetterByIndex: (index: number) => string;
+  isPetitioner: boolean;
+  isPractitioner: boolean;
```

Comment:
Is it possible for for both `isPetitioner` and `isPractitioner` to both be true (or both be false)?  If not, would it be better to just have one of these vars?

Reply:
It’s not possible for both isPetitioner and isPractitioner to be true or both false at the same time, so we could simplify this by having just one property and negating it when necessary. Originally, I used both properties for readability, but I do see that we are negating in some places. Are you okay with this change going in a followup PR along with the cleanup?

Reply:
Thanks Nechama, yeah I think a followup PR is fine.

Reply:
Oh and I do like the readability argument, my only concern is us winding in a situation where we wind up in an invalid state.  If practical, I think it would be fine to have both props, but have one be a computed prop that is the negation of the other.